src/olympia/shelves/views.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions, viewsets
+
+from olympia.shelves.models import ShelfManagement
+from olympia.shelves.serializers import HomepageSerializer
+
+
+class ShelfViewSet(viewsets.ModelViewSet):
+    queryset = ShelfManagement.objects.all()
```

Comment:
So, really here we want a queryset of Shelf, but filtered and ordered by some of the ShelfManagement properties.  Something like `Shelf.objects.filter(shelfmanagement__enabled=True).order_by('shelfmanagement__position')`